.gitignore

```diff
@@ -12,3 +12,4 @@ target/
 
 # Ignore all generic character images
 res/images/characters/generic/
+/tail.txt
```

Comment:
I guess, this can be removed?

Reply:
Yes, that shouldn't be there. Removed it now.